tools/doc/html.js

```diff
@@ -66,10 +66,18 @@ const gtocHTML = unified()
 const templatePath = path.join(docPath, 'template.html');
 const template = fs.readFileSync(templatePath, 'utf8');
 
-function wrapSections(content) {
+function processContent(content) {
+  content = content.toString();
+  // Increment header tag levels to avoid multiple h1 tags in a doc.
+  // This means we can't already have an <h6>.
+  if (content.includes('<h6>')) {
+    throw new Error('Cannot increment a level 6 header');
+  }
```

Comment:
In case someone adds attributes headers later, let's have an Error thrown to make sure the regex will be updated to take it into account.
```suggestion
  }
  if (/<h[1-5][^>]/.test(content) {
    throw new Error('Cannot parse header tag with attributes');
  }
```